docs/apache-airflow-providers-amazon/operators/emr_eks.rst

```diff
@@ -0,0 +1,87 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+.. _howto/operator:EMRContainersOperators:
+
+Amazon EMR on EKS Operators
+===========================
+
+.. contents::
+  :depth: 1
+  :local:
+
+Prerequisite Tasks
+------------------
+
+.. include:: _partials/prerequisite_tasks.rst
+
+Overview
+--------
+
+Airflow to Amazon EMR on Amazon EKS integration provides a way to run Apache Spark jobs on Kubernetes.
+
+- :class:`~airflow.providers.amazon.aws.operators.emr_containers.EMRContainerOperator`
+
+There is an example dag that shows how to run a job with this operator.
+
+- example_emr_eks_job.py
```

Comment:
```suggestion
```
I know that some AWS guides contain references to DAG example, but it is not necessary. This filename itself has no value. We at least need a link to the file to be able to open this file.

On the other hand, each example uses the exampleinclude directive, which means we have a link to the source code.